building_energy_standards_data/applications/create_openstudio_standards_json.py

```diff
@@ -362,13 +367,15 @@ def create_openstudio_standards_code_version_data_json(
     code_version: str,
     template: str,
     tables_to_export: dict,
+    osstd_repository_path: str,
```

Comment:
It might be better to provide a default to the `osstd_repository_path = './'` argument here.
Part of the reason is because this function is also exposed. So external application could calling this function.
Another approach is the same as the prev. comment, add a safe guard to handle the exception.